ndb/tests/unit/test_model.py

```diff
@@ -2283,16 +2295,117 @@ def test_constructor():
 
 class TestMetaModel:
     @staticmethod
-    def test_constructor():
-        with pytest.raises(NotImplementedError):
-            model.MetaModel()
+    def test___repr__():
+        expected = "Model<>"
+        assert repr(model.Model) == expected
+
+    @staticmethod
+    def test___repr__extended():
+        class Mine(model.Model):
+            first = model.IntegerProperty()
+            second = model.StringProperty()
+
+        expected = (
+            "Mine<first=IntegerProperty('first'), "
+            "second=StringProperty('second')>"
+        )
+        assert repr(Mine) == expected
+
+    @staticmethod
+    def test_bad_kind():
```

Comment:
ISTM that this should be in `TestMetaModel` since the behavior of `MetaModel` invokes `Model._fix_up_properties` to actually cause this failure.